### PR TITLE
fix: move to use mavencentral instead of jcenter

### PR DESCRIPTION
**Pull request by maxandersen**
Please delete this text, and add description of the topic solved by this PR.

To make releases as automated as possible we use conventional commits formats.
Thus commits and pull-requests titles should before merge follow the Conventional Commits spec.

Details in https://github.com/zeke/semantic-pull-requests

If in doubt then open the pull-request and we'll help you - Thank you!
-->

### CODE REVIEW

**Comment by maxandersen** [comment]
the extra commit is due to missing merge from tag - will be fixed when merging this

File: src/main/java/dev/jbang/dependencies/DependencyUtil.java [hunk 1]
```diff
@@ -39,6 +39,7 @@
 
 	static {
 		aliasToRepos = new HashMap<>();
+		aliasToRepos.put("mavencentral", "https://repo1.maven.org/maven2/");
 		aliasToRepos.put("jbossorg", "https://repository.jboss.org/nexus/content/groups/public/");
 		aliasToRepos.put("redhat", "https://maven.repository.redhat.com/ga/");
 		aliasToRepos.put("jcenter", "https://jcenter.bintray.com/");
```
Comment (quintesse) [inline src/main/java/dev/jbang/dependencies/DependencyUtil.java:45]: this one you're keeping for now?

Comment (maxandersen) [reply src/main/java/dev/jbang/dependencies/DependencyUtil.java:45]: Yeah. Don't break those who used it explicitly.

Comment (quintesse) [reply src/main/java/dev/jbang/dependencies/DependencyUtil.java:45]: Yeah. Should we perhaps show a warning to people using it?

Comment (maxandersen) [reply src/main/java/dev/jbang/dependencies/DependencyUtil.java:45]: not sure where to best put that - we can do that in separate PR if we really feel for it.

Comment (quintesse) [reply src/main/java/dev/jbang/dependencies/DependencyUtil.java:45]: Perhaps right where we parse //REPOS?

**Pull request merged**